src/styles.js

```diff
@@ -43,6 +43,11 @@ color.propTypes = {
   ...bgColor.propTypes,
 }
 
+export const opacity = responsiveStyle({
+  prop: 'opacity',
+  alias: 'o',
```

Comment:
I think the shorthand should be removed; the existing shorthands are mostly meant for backwards compatibility, and I can't imagine `opacity` being used nearly enough to warrant a new shorthand

Reply:
@jxnblk @jsonmaur I think we should add `key: 'opacities'`, so instead of theming as `opacity: [0, 0.5, 1]` we would theme it as `opacities: [0, 0.5, 1]`. The plural form is the pattern Styled System follows for theming.